pkg/bundle/bundle.go

```diff
@@ -65,12 +65,13 @@ type LocationRef struct {
 
 // BaseImage contains fields shared across image types
 type BaseImage struct {
-	ImageType string         `json:"imageType" mapstructure:"imageType"`
-	Image     string         `json:"image" mapstructure:"image"`
-	Digest    string         `json:"digest,omitempty" mapstructure:"digest"`
-	Size      uint64         `json:"size,omitempty" mapstructure:"size"`
-	Platform  *ImagePlatform `json:"platform,omitempty" mapstructure:"platform"`
-	MediaType string         `json:"mediaType,omitempty" mapstructure:"mediaType"`
+	ImageType     string         `json:"imageType" mapstructure:"imageType"`
+	Image         string         `json:"image" mapstructure:"image"`
+	OriginalImage string         `json:"originalImage" mapstructure:"originalImage"`
```

Comment:
In the meantime, we also mergefd #704, which pulls out `pkg/bundle` into a separate directory.

We should make a PR to https://github.com/deislabs/cnab-go with the updated bundle structure, rebase `master` again and pinpoint a version of the Go package.

Let me know if you want me to update `cnab-go` and eventually have a first 0.x release we can use here.

I apologize, I should have spotted this last week.

Reply:
I'll absorb the merge of https://github.com/deislabs/duffle/pull/704 here.

I think we can defer updating to `cnab-go` and pick that up in due course.

Reply:
https://github.com/deislabs/cnab-go/pull/4 raised.

Reply:
This PR picked up https://github.com/deislabs/cnab-go/pull/4.